static/app/components/events/eventTagsAndScreenshot/index.tsx

```diff
@@ -115,16 +117,29 @@ function EventTagsAndScreenshots({
 
   return (
     <Wrapper showScreenshot={showScreenshot} showTags={showTags}>
+      <TagWrapper hasEventContext={hasEventContext}>
+        {hasEventContext && (
+          <TagsHighlightWrapper>
+            <TagsHighlight event={event} />
+          </TagsHighlightWrapper>
+        )}
+        {showTags && (
+          <Tags
+            organization={organization}
+            event={event}
+            projectSlug={projectSlug}
+            location={location}
+          />
+        )}
+      </TagWrapper>
       {showScreenshot && (
         <div>
           <ScreenshotWrapper>
             <ScreenshotDataSection
               data-test-id="screenshot-data-section"
               title={
                 screenshots.length > 1
-                  ? tct('[current] of [total] [link:screenshots]', {
-                      current: screenshotInFocus + 1,
-                      total: screenshots.length,
+                  ? tct('[link:Screenshots]', {
```

Comment:
Can we sneak in this change in here as well? Making this and the screenshot link in line 145 blue so it's obvious it's clickable? 
![Screen Shot 2022-11-16 at 11 11 26 AM](https://user-images.githubusercontent.com/63818634/202233448-e059fed3-5a8f-43c3-a6c5-88002f1017bd.png)

Reply:
https://github.com/getsentry/sentry/pull/41337/commits/6ae9e58f1eb9594b55a69ef29a65373aee8c06b5

Added! I also simplified the screenshot/screenshots ternary for the title prop by just using `tn()` inside the link